### PR TITLE
Archive logic

### PR DESCRIPTION
**Pull request by saifudinbadani**
Archive logic and component added.

### CODE REVIEW

File: src/components/ArchivePage/Archive.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { useNotes } from '../../context/NotesContext';
+import '../HomePage/Homepage.css'; 
+
+ export const Archive = () => {
+
+    const { noteState: { archiveNotes}, noteDispatch } = useNotes();
+
+    
+    console.log(archiveNotes)
```
Comment (devsurajrai) [inline src/components/ArchivePage/Archive.js:9]: please remove this console.log

File: src/components/Navigation/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    <h1 to='/' className="nav-link font-size-2pt6 heading-1">Jotter</h1>
+    <div className="input-search pos-rltv m-rght-auto">
+        <i className="fas fa-filter pos-absolute font-size-2pt6"></i>
+        <input type="text" placeholder="search" className="nav-input-search"/>
+    </div>
+    {/* <Link to='/productlist' className="nav-link-cta">Shop Now</Link>
+    <Link to='/login' className="nav-link"><button
+            className="btn btn-solid-secondary">Login</button></Link> */}
+    </nav>
+}
```
Comment (devsurajrai) [inline src/components/Navigation/Navigation.js:16]: remove this commented code if not needed.

**Review by devsurajrai — COMMENTED** [review]
Everything Looks Good. Just remove the unnecessary code as suggested.

**Review by vaishnavipaliya11 — APPROVED** [review]
Remove commented code. Rest all looks clean.

**Pull request merged**